src/test/java/com/endava/TestCreativeAndUXPage.java

```diff
@@ -15,53 +15,167 @@
 
 public class TestCreativeAndUXPage {
 
-	private static final Logger logger = LoggerFactory.getLogger(TestCreativeAndUXPage.class);
+    private static final Logger logger = LoggerFactory.getLogger(TestCreativeAndUXPage.class);
 
-	private HomePage homePage;
-	private CreativeAndUXPage creativeAndUXPage;
+    private HomePage homePage;
+    private CreativeAndUXPage creativeAndUXPage;
 
-	@BeforeTest
-	@Parameters({ "browser" })
-	public void setUp(String browser) {
-		WebDriverWrapper.setUpDriver(browser);
-	}
+    @BeforeTest
+    @Parameters({"browser"})
+    public void setUp(String browser) {
+        WebDriverWrapper.setUpDriver(browser);
+    }
 
-	@BeforeMethod
-	@Parameters({ "browser" })
-	public void openBrowser(String browser) {
-		homePage = new HomePage(WebDriverWrapper.createDriver(browser));
-		homePage.open();
-	}
+    @BeforeMethod
+    @Parameters({"browser"})
+    public void openBrowser(String browser) {
+        homePage = new HomePage(WebDriverWrapper.createDriver(browser));
+        homePage.open();
+    }
+
+    /**
+     * Go to endava.com. Scroll down to the page footer and from the "Services"
```

Comment:
Please be consistent in the way the test header is written. Any logic is fine, but we need to have all the test cases written in the same manner (regarding new lines and numbering), i.e.

1. Go to endava.com
2. Scroll down...
3. Click on ...

Reply:
Done.